src/main/services/api.ts

```diff
@@ -0,0 +1,64 @@
+import axios from 'axios';
+
+export default class Api {
+  private baseUrl = process.env.API_URL;
+  private apiKey = process.env.API_KEY;
+  private headers;
+
+  constructor(baseUrl?: string, apiKey?: string) {
+    if (baseUrl) {
+      this.baseUrl = baseUrl;
+    }
+    else if (!this.baseUrl) {
+      throw new Error('No base url supplied and no default set in environment variables');
+    }
+
+    if (apiKey) {
+      this.apiKey = apiKey;
+    }
+    else if (!this.apiKey) {
+      throw new Error('No api key supplied and no default set in environment variables');
+    }
+
+    this.headers = {
+      'Content-Type': 'application/json',
+      'Authorization': `Bearer ${this.apiKey}`,
+      'Accept': 'application/json',
+      'UserId': ''
```

Comment:
I think this header is `X-User-Id?`
```suggestion
      'X-User-Id': ''
```